src/schemes/fluid/weakly_compressible_sph/state_equations.jl

```diff
@@ -49,3 +49,50 @@ function inverse_state_equation(state_equation::StateEquationCole, pressure)
 
     return reference_density * tmp^(1 / exponent)
 end
+
+@doc raw"""
+    StateEquationIdealGas(;sound_speed, reference_density, gamma, background_pressure=0.0,
+                           clip_negative_pressure=false)
+
+Equation of state to describe the relationship between pressure and density
+of a gas using the Ideal Gas Law.
+
+# Keywords
+- `sound_speed`            : Artificial speed of sound.
+- `reference_density`      : Reference density of the fluid.
+- `gamma`                  : Heat-capacity ratio
+- `background_pressure=0.0`: Background pressure.
```

Comment:
Can you please add `clip_negative_pressure` here in the list as well? And it seems that is also missing in the SE by Cole.